python/lsst/ap/association/association.py

```diff
@@ -129,13 +129,36 @@ def run(self,
 
         diaObjects = diaObjects.append(matchResult.new_dia_objects,
                                        sort=True)
+        # if len(diaObjects) > 0:
+        #     dups = diaObjects.iloc[[0, -1]]
+        #     diaObjects = diaObjects.append(dups, sort=True)
```

Comment:
Delete this commented-out code.

Reply:
Thanks for the catch. Was keeping this in until the decision was made on warn vs raise for finding duplicates. Will take it out.